scripts/get_tox_envs.py

```diff
@@ -52,9 +54,13 @@ def get_changed_tox_envs(all_envs: bool, travis_strict: bool, plan: bool) -> Non
     unique_directories = list(unique_directories)
     # If we only have one and it's a valid tox environment, run that one
     if len(unique_directories) == 1 and (unique_directories[0] in default_environments):
```

Comment:
Unrelated comment. What if two tutorials have changed? Why not run those two instead of everything?